wit/deps/golem-rpc/wasm-rpc.wit

```diff
@@ -88,15 +105,16 @@ interface types {
   }
 
   resource wasm-rpc {
```

Comment:
Does it make sense to call this resource golem-rpc? I don't think the is really much 'wasm' about it

Reply:
I'm trying to do the minimum amount of changes, and we called this particular implementation of RPC as "wasm-rpc" since the beginning